### PR TITLE
Add ability to specify lanIp argument (fix #27)

### LINKED ISSUE #27: lanIp is a virtual adapter IP
When running cordova-serve, then deploying on a device, the server IP used is something like 169.254... (which is the IP assigned to a virtual adapter in windows). 
Digging into vue-cli-plugin-cordova lead me to this line: https://github.com/dekimasoon/vue-cli-plugin-cordova/blob/3a958fd5506e5cfc9c3e7540aef2f8118b9ac4cf/index.js#L34
Is it possible to consider adding an option specifying which IP to use, bypassing the one returned by `address.ip()` ?
I don't think I'm the only one who have virtual machines, and I didn't find any similar issue.

Thanks in advance for any help / answer.

### PR DESCRIPTION
**Pull request by tartinesKiller**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -31,8 +33,9 @@ module.exports = (api, options) => {
         host: args.host || process.env.HOST || projectDevServerOptions.host || defaults.host,
         port,
         https: args.https || projectDevServerOptions.https || defaults.https,
-        lanIp: address.ip()
+        lanIp: args["lan-ip"] || defaults.lanIp
```
Comment (dekimasoon) [inline index.js]: Please use single quotation.

Comment (tartinesKiller) [reply index.js]: Done

File: redirect.ejs [hunk 2]
```diff
@@ -19,6 +19,9 @@
         var protocol = '<%- https ? "https" : "http" %>'
         var port = '<%- port %>'
         var platform = device.platform.toLowerCase()
+        if(platform === 'amazon-fireos') {
```
Comment (dekimasoon) [inline redirect.ejs]: Please put a space before the parentheses.
And I think we should add some comments to explain why this branch is necessary.

Comment (tartinesKiller) [reply redirect.ejs]: Added, I hope it's not too verbose!

**Comment by dekimasoon** [comment]
I just released your updates as v1.0.0-beta.8. Thanks!

**Comment by tartinesKiller** [comment]
Great! Thanks to you, happy to help

**Pull request merged**